spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:all) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
```

Comment:
You made great efforts in making this project. However, there are some problems to fix.

- [ ] You have two failing tests:


 
<img width="900" alt="Screen Shot 2022-09-16 at 15 36 14" src="https://user-images.githubusercontent.com/69011963/190620970-be350679-b485-4c00-bde3-260ca6579af4.png">

Kindly fix them. To make your code approved. Your all tests must pass as expected.